packages/std/src/query/wasm.rs

```diff
@@ -8,7 +8,8 @@ use crate::Binary;
 #[serde(rename_all = "snake_case")]
 pub enum WasmQuery {
     /// this queries the public API of another contract at a known address (with known ABI)
-    /// return value is whatever the contract returns (caller should know)
+    /// Return value is whatever the contract returns (caller should know), wrapped in a
+    /// ContractResult that is JSON encoded.
```

Comment:
This comment update changes schema in `contracts/reflect`.
I wanted to fix it myself, but appears that I don't have access right. ;_;